projects/dex-ui/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Should extend root tsconfig and overwrite only what's needed

Reply:
👍